server/views/components/actions-card/actionsCard.test.ts

```diff
@@ -9,28 +9,129 @@ nunjucks.configure([
   __dirname,
 ])
 describe('Tests for actions card component', () => {
-  it('Should display a link for each action', () => {
-    const actions: Action[] = [
-      { title: 'Do a thing', href: '/thing' },
-      { title: 'Do another thing', href: '/another-thing' },
-    ]
-    const content = nunjucks.render('test.njk', { actions })
-    const $ = cheerio.load(content)
-    const actionListItems = $('.actions-list-list').find('li')
-    expect(actionListItems.length).toStrictEqual(2)
-    expect($(actionListItems[0]).find('a').text()).toStrictEqual('Do a thing')
-    expect($(actionListItems[0]).find('a').attr('href')).toStrictEqual('/thing')
-    expect($(actionListItems[1]).find('a').text()).toStrictEqual('Do another thing')
-    expect($(actionListItems[1]).find('a').attr('href')).toStrictEqual('/another-thing')
+  describe('Core action card component', () => {
+    it('Should render the action card component given actions', () => {
+      const actions: Action[] = [
+        { title: 'Do a thing', href: '/thing' },
+        { title: 'Do another thing', href: '/another-thing' },
+      ]
+      const params = { actions }
+
+      const content = nunjucks.render('test.njk', { params })
+
+      const $ = cheerio.load(content)
+      const actionCard = $('.actions-card')
+      expect(actionCard.length).toEqual(1)
+    })
+
+    it('Should render the action card component given a specific ID', () => {
+      const actions: Action[] = [
+        { title: 'Do a thing', href: '/thing' },
+        { title: 'Do another thing', href: '/another-thing' },
+      ]
+      const params = {
+        id: 'my-actions-card',
+        actions,
+      }
+
+      const content = nunjucks.render('test.njk', { params })
+
+      const $ = cheerio.load(content)
+      const actionCard = $('#my-actions-card')
+      expect(actionCard.length).toEqual(1)
+    })
+
+    it('Should support setting custom attributes such as data-qa', () => {
+      const actions: Action[] = [
+        { title: 'Do a thing', href: '/thing' },
+        { title: 'Do another thing', href: '/another-thing' },
+      ]
+      const params = {
+        actions,
+        attributes: { 'data-qa': 'my-actions-card' },
+      }
```

Comment:
Can we type up an `ActionCardParams` or something similar for this top level type? I know we're not directly using the types as this one is just inlined in the template but it would be nice for the tests to be documentation for the component and constrained using the types.